compose-builder/docker-compose-base.yml

```diff
@@ -157,6 +157,23 @@ services:
     security_opt: 
       - no-new-privileges:true
 
+  common-config:
+    image: ${CORE_EDGEX_REPOSITORY}/core-common-config-bootstrapper${ARCH}:${CORE_EDGEX_VERSION}
+    user: "${EDGEX_USER}:${EDGEX_GROUP}"
+    container_name: edgex-core-common-config-bootstrapper
+    hostname: edgex-core-common-config-bootstrapper
+    read_only: true
+    networks:
+      - edgex-network
+    env_file:
+      - common.env
+    environment:
+      EDGEX_SECURITY_SECRET_STORE: false
```

Comment:
could be removed: this already in common.env

Reply:
Good catch!

Reply:
Removed